libopae/src/sysfs.c

```diff
@@ -29,13 +29,17 @@
 #endif // HAVE_CONFIG_H
```

Comment:
Looks like you ran a formatter on this file.  I can't tell which changes are required for your object API.

Reply:
😢